src/pybind/mgr/cephadm/module.py

```diff
@@ -1905,7 +1905,7 @@ def get_minimal_ceph_conf(self) -> str:
         })
         extra = self.extra_ceph_conf().conf
         if extra:
-            config += '\n\n' + extra.strip() + '\n'
+            config += extra.strip().replace('[global]','') + '\n'
```

Comment:
This probably works for most normal use cases. There's a couple details I'm thinking could cause an issue though

1. Technically you can have a ceph conf with a global section not first I believe. Something like
```
[mon]
k = v

[global]
k2 = v2
```
That wouldn't work if set as the extra ceph conf with this patch as the `k2 = v2` setting would end up in the [mon] section.

2.  I think it's possible in the future that the minimal ceph conf could include sections that aren't [global]
3. If there is an issue here we are fixing we'd ideally have some test cases specified so that we know what works.
4. Currently make check is failing on this PR as this change caused one of the unit tests to fail

I created a quick sample commit that tries to formalize the combining of the minimal ceph conf and the extra ceph conf that handles the points above to generally show what I mean https://github.com/adk3798/ceph/commit/b905bab6e94568d33a83c9c54c95500102dc1e03. Thoughts?

Reply:
Thanks for your quick response. I only considered case which current minimal conf is. Yes it will cause issues according to your sample. And I think you have pointed the reason why make check failed( \n\n). I will fix this soon.

Reply:
If you can, you can do it. I haven't cleaned it up and you can maybe make sure it works with whatever caused you to hit the issue in the first place.

Reply:
Thanks so much. It works fine in my cases, please review again.